tests/src/performance/Scenario/JitBench/JitBench.csproj

```diff
@@ -54,5 +54,8 @@
           Overwrite="true"
           Encoding="Unicode"/>
   </Target>
+  <Target Name="AfterBuild">
+      <Copy SourceFiles="Resources\word2vecnet.patch" DestinationFolder="$(OutDir)" ContinueOnError="true" />
```

Comment:
Don't you want to fail the build if not copied?

Reply:
removed.

Reply:
@dotnet-bot test Windows_NT x64 perf scenarios please

Reply:
removed.